src/transformers/models/gemma/configuration_gemma.py

```diff
@@ -1,5 +1,12 @@
+#           🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨🚨
+#               This file was automatically generated from <path_to_diff_file.py>.
```

Comment:
Is it supposed to be a real path to a diff file or just `<path_to_diff_file.py>`?

Reply:
just `<path_to_diff_file.py>` but we can also add the real path